src/index.html

```diff
@@ -11,8 +11,58 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute should link to a CSS file, not an SCSS file. Please ensure you are linking to the compiled CSS file, such as './styles/index.css'.